Insights/Account/UEBAAnomaly.yaml

```diff
@@ -0,0 +1,88 @@
+      - ProjectedName: Uncommon Among Peers
+        Query: "{{BaseQuery}} | {{RowFilter}} and (Evidence.ActionUncommonlyPerformedAmongPeers == true)"
+      - ProjectedName: Uncommon in Tenant
+        Query: "{{BaseQuery}} | {{RowFilter}} and (Evidence.ActionUncommonlyPerformedInTenant == true)"
+      - ProjectedName: First Time Tenant Anomaly
+        Query: "{{BaseQuery}} | {{RowFilter}} and (Evidence.FirstTimeActionPerformedInTenant == True)"
+    - Filter: " where (ActionType in('Sign-in') and  (ActivityInsights.FirstTimeUserConnectedFromCountry == True or  ActivityInsights.FirstTimeConnectionFromCountryObservedInTenant == True or ActivityInsights.CountryUncommonlyConnectedFromInTenant == True or ActivityInsights.CountryUncommonlyConnectedFromAmongPeers == True))"
+      Summarize: summarize ['First Time User Anomaly']=countif(Evidence.FirstTimeUserConnectedFromCountry == True),['Uncommon Among Peers']=countif(Evidence.CountryUncommonlyConnectedFromAmongPeers == True), ['Uncommon in Tenant']=countif(Evidence.CountryUncommonlyConnectedFromInTenant == True) , ['First Time Tenant Anomaly']=countif(Evidence.FirstTimeConnectionFromCountryObservedInTenant == True)
+      Project: project Anomaly = 'Anomloaus location', ['First Time User Anomaly'], ['Uncommon Among Peers'], ['Uncommon in Tenant'], ['First Time Tenant Anomaly']
+      LinkColumnsDefinitions:
+      - ProjectedName: First Time User Anomaly
+        Query: "{{BaseQuery}} | {{RowFilter}} and (Evidence.FirstTimeUserConnectedFromCountry == True)"
+      - ProjectedName: Uncommon Among Peers
+        Query: "{{BaseQuery}} | {{RowFilter}} and (Evidence.CountryUncommonlyConnectedFromAmongPeers == True)"
+      - ProjectedName: Uncommon in Tenant
+        Query: "{{BaseQuery}} | {{RowFilter}} and (Evidence.CountryUncommonlyConnectedFromInTenant == True)"
+      - ProjectedName: First Time Tenant Anomaly
+        Query: "{{BaseQuery}} | {{RowFilter}} and (Evidence.FirstTimeConnectionFromCountryObservedInTenant == True)"
+    - Filter: " where (ActionType in('InteractiveLogon','Sign-in','ResourceAccess') and (ActivityInsights.FirstTimeDeviceObservedInTenant == True or ActivityInsights.FirstTImeUserConnectedFromDevice == True or ActivityInsights.DeviceUncommonlyUsedAmongPeers == True or ActivityInsights.FirstTimeResourceAccessedInTenant == True or ActivityInsights.ResourceUncommonlyAccessedAmongPeers == True or ActivityInsights.FirstTimeUserAccessedResource == True))"
+      Summarize: summarize ['First Time User Anomaly']=countif(Evidence.FirstTImeUserConnectedFromDevice == True or Evidence.FirstTimeUserAccessedResource == True),['Uncommon Among Peers']=countif(Evidence.DeviceUncommonlyUsedAmongPeers == True or Evidence.ResourceUncommonlyAccessedAmongPeers == True), ['Uncommon in Tenant']=0 , ['First Time Tenant Anomaly']=countif(Evidence.FirstTimeDeviceObservedInTenant == True or Evidence.FirstTimeResourceAccessedInTenant == True)
```

Comment:
Need to add 'by UPN' at the end, otherwise when we don't have a match, we will get rows with 0 returns which is not how we want no results to show up in the UX.